### PR TITLE
Config cleanups

### PR DESCRIPTION
**Pull request by sparticlesteve**
Proposed updates to simplify and clean up the configs.

- `base` now corresponds to our tutorial baseline config at bs 64, full training
- `short` is for single-gpu testing and profiling, with fewer samples and epochs
- `short_opt` is `short` with all optimizations, for convenience
- `bs64_opt` is `base` with all optimizations
- `bsXXX_opt` is batch size XXX with optimizations

This makes the commands look like:

```bash
# Single GPU training - full
sbatch -n 1 ./submit_pm.sh

# Short config for testing/profiling
sbatch -n 1 ./submit_pm.sh --config=short

# Benchy run
sbatch -n 1 ./submit_pm.sh --config=short --enable_benchy

# Baseline profiling
ENABLE_PROFILING=1 PROFILE_OUTPUT=baseline \
    sbatch -n 1 ./submit_pm.sh --config=short --enable_manual_profiling

# Running with all optimizations
sbatch -n 1 ./submit_pm.sh --config=short \
    --data_loader_config=dali-lowmem --enable_amp --enable_apex --enable_jit

# Running with all optimizations using opt config
sbatch -n 1 ./submit_pm.sh --config=short_opt

# Full training, single-gpu, optimized
sbatch -n 1 ./submit_pm.sh --config=bs64_opt

# Single-node distributed training
sbatch -n 4 ./submit_pm.sh --config=bs256_opt

# 32 GPU training
sbatch -n 32 ./submit_pm.sh --config=bs2048_opt
```

### CODE REVIEW

File: config/UNet.yaml [hunk 1]
```diff
@@ -38,71 +41,46 @@ base: &base
...
   num_epochs: 160
 
-bs8192: # 128 GPUs
-  <<: *bs64
+bs8192_opt: # 128 GPUs
+  <<: *bs64_opt
   global_batch_size: 8192
   num_epochs: 640
-
-bs8192_short:
```
Comment (pzharrington) [inline config/UNet.yaml:109]: Can we leave this one in, or at least make `num_epochs=320` the default for `bs8192`? With this big a batchsize the 640 epoch run can diverge/go nan, whereas my test with `num_epochs=320` did complete

Comment (sparticlesteve) [reply config/UNet.yaml:109]: ah, yeah, sure. I thought that was maybe just one for testing.

Comment (pzharrington) [reply config/UNet.yaml:109]: Cool, otherwise it looks good to me.

Comment (sparticlesteve) [reply config/UNet.yaml:109]: I adjusted bs8192_opt to use 320 epochs and added a warning there about training stability

**Comment by sparticlesteve** [comment]
I've now updated README example commands to use the new config names.

**Pull request merged**